docs/source/markdown/podman-create.1.md

```diff
@@ -1224,6 +1224,13 @@ The default working directory for running binaries within a container is the roo
 The image developer can set a different default with the WORKDIR instruction. The operator
 can override the working directory by using the **-w** option.
 
+#### **\-\-pidfile**=*path*
+
+Write the pid of the container process to a file.
+
+The default pidfile is RunDir/pidfile.
```

Comment:
I had a hard time translating this to the location of the pidfile.  Since we don't document the location of the conmon pidfile, I am not sure this is helpful to the user?  When you set this flag, does it store the pidfile in two locations?

Reply:
After this is done, will I be able to inspect the location of the pidfile?

 podman inspect -l  | grep -i pidfile
        "ConmonPidFile": "/run/user/3267/containers/overlay-containers/7e0e1fadac3f047aa7c880c694a93186768feab9a2198d0f620a01706094b428/userdata/conmon.pid",

Reply:
If you are hiding this from remote, then it should be documented that it is not supported for remote.

Reply:
I write this docoment according to https://github.com/containers/podman/blob/master/docs/source/markdown/podman-create.1.md#--conmon-pidfilepath.
When set this flag, it stores the pidfile in one location.

Reply:
After this is done, the container‘s inspection will show the pidfile location in CreateCommand, not a property. Is it ok?

podman inspect -l | grep -i pidfile
"CreateCommand": [
                "podman",
                "create",
                "--pidfile",
                "/root/a.txt",

Reply:
Understood, but this line:

 The default pidfile is RunDir/pidfile. 

Is not really helpful, or at least forced me to do detectivework.  As a user, I have no idea where RunDir is?

Something like: to find the default location use:
```
You can find the default location for this file by executing the following command on the contaner.
$ podman inspect --format '{{ .ConmonPidFile }}' $CID
/run/user/1000/containers/overlay-containers/7e0e1fadac3f047aa7c880c694a93186768feab9a2198d0f620a01706094b428/userdata/conmon.pid
```

Reply:
You need to add the Inspection of PidFile to match conmonpidfile,

Reply:
Added detail description for pidfile. Added description about how to find the pidfile location inspect command.
And documented the options conmon-pidfile and pidfile are not support on remote.